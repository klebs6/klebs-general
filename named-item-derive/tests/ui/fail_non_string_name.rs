// named-item-derive/tests/trybuild/fail_non_string_name.rs

#![allow(unused)]
use named_item_derive::NamedItem;
use named_item::*;

#[derive(NamedItem)]
pub struct WrongNameField {
    name: i32,  // not `String`
}

fn main() {}

