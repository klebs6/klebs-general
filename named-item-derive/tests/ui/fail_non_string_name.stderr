error: `name` field must be `String`
 --> tests/ui/fail_non_string_name.rs:9:11
  |
9 |     name: i32,  // not `String`
  |           ^^^
